es-vue-base/src/lib-components/EsPagination.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <b-pagination
+        v-bind="$attrs"
+        class="mb-0"
+        first-number
+        last-number
+        page-class="page-number font-size-sm mx-25 mx-sm-50"
+        ellipsis-class="ellipses mx-25 mx-sm-50"
+        prev-class="prev-next"
+        next-class="prev-next"
+        :aria-controls="listID"
+        v-on="$listeners">
+        <template #prev-text>
+            <IconChevronLeft />
+        </template>
+        <template #next-text>
+            <IconChevronRight />
+        </template>
+    </b-pagination>
+</template>
+
+<script>
+import { BPagination } from 'bootstrap-vue';
+import IconChevronLeft from '../lib-icons/icon-chevron-left.vue';
+import IconChevronRight from '../lib-icons/icon-chevron-right.vue';
+
+export default {
+    name: 'EsPagination',
+    components: {
+        BPagination,
+        IconChevronLeft,
+        IconChevronRight,
+    },
+    props: {
+        listId: {
+            type: String,
+            required: true,
+        },
+    },
+};
+</script>
+
+<style lang="scss">
+@import '~@energysage/es-bs-base/scss/includes';
+
+.pagination {
+    .page-number, .prev-next, .ellipses {
+        .page-link {
+            background-color: transparent;
+            color: $dark;
+            border: 0;
+        }
+    }
+
+    .page-number {
+        .page-link {
+            height: 2.25rem;
+            width: 2.25rem;
+
+            &:hover {
+                background-color: transparent;
+                color: $black;
+                font-weight: bold;
+            }
+        }
+
+        &.active {
+            .page-link {
+                background: linear-gradient($white, $white) padding-box,
+                    linear-gradient(to bottom right, #f6b140, #c84486) border-box;
```

Comment:
Be great if these hex code were vars; one already is. Maybe there's a [$pink ](https://github.com/EnergySage/es-ds/blob/6a7f9bc2ee6fd3bba1be8e6d88bf7236daa2ad44/es-bs-base/scss/_variables.scss#L111) that is "close enough".

`f6b140 = $yellow-500`